models/migrations/v1_22/v283_test.go

```diff
@@ -7,22 +7,22 @@ import (
 	"testing"
 
 	"code.gitea.io/gitea/models/migrations/base"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func Test_AddCombinedIndexToIssueUser(t *testing.T) {
 	type IssueUser struct {
-		UID     int64 `xorm:"INDEX unique(uid_to_issue)"` // User ID.
-		IssueID int64 `xorm:"INDEX unique(uid_to_issue)"`
+		ID          int64 `xorm:"pk autoincr"`
+		UID         int64 `xorm:"INDEX"` // User ID.
```

Comment:
These changes are unlated?

Reply:
Related. The old code is not right, and causes CI failures.

I guess the test system has many problems, but I haven't got time to look into.